cpu/cortexm_common/include/cpu.h

```diff
@@ -68,6 +68,30 @@ void cpu_init(void);
  */
 void cortexm_init(void);
```

Comment:
Add a note in the doxygen for cortexm_init that it will call init fpu, prio and misc. To be clear that CPU port developers only need to call the individual functions if the default init is not working.